flocker/provision/_install.py

```diff
@@ -282,6 +275,57 @@ def task_create_flocker_pool_file():
     ])
 
 
+def task_install_zfs(distribution, variants=set()):
+    commands = []
+    if distribution == 'ubuntu-14.04':
```

Comment:
That separate issue is probably https://clusterhq.atlassian.net/browse/FLOC-1670.